types/swagger2openapi/index.d.ts

```diff
@@ -0,0 +1,145 @@
+// Type definitions for swagger2openapi 7.0
+// Project: https://github.com/Mermade/oas-kit/tree/main/packages/swagger2openapi
+// Definitions by: rxliuli <https://github.com/rxliuli>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { OpenAPIV2, OpenAPIV3 } from 'openapi-types';
+import { Readable } from 'node:stream';
+
+export class S2OError extends Error {}
+
+export interface ConvertBaseOptions {
+    text: string; // Both If not already a truthy value, will be set to the input text of the conversion
+}
+
+export interface ConvertInputOptions extends ConvertBaseOptions {
+    agent: object; // Input Optional http(s).Agent to be used when fetching resources
```

Comment:
Are you intending for these comments to be visible to users? If so, you may want to make these proper JSDoc comments above, like:

```suggestion
    /**
     * Input Optional http(s).Agent to be used when fetching resources
     */
    agent: object;
```